src/mon/OSDMonitor.cc

```diff
@@ -8491,88 +8490,73 @@ bool OSDMonitor::prepare_command_impl(MonOpRequestRef op,
       return prepare_set_flag(op, CEPH_OSDMAP_NODEEP_SCRUB);
     else if (key == "notieragent")
       return prepare_set_flag(op, CEPH_OSDMAP_NOTIERAGENT);
-    else if (key == "sortbitwise") {
+    else if (key == "sortbitwise" ||
```

Comment:
i don't really think this improves the readability. at the first glance, it's good to minimize the duplicated code, but it actually complicates the code.